File: Python/Ejercicio 4/main.py
negativos = 0
ceros = 0
positivos = 0

for i in range(10):
    nua = int(input("ingrese un numero"))
    if (nua < 0):
        negativos = negativos + 1
    if (nua == 0):
        ceros = ceros + 1
    if (nua > 0):
        positivos = positivos + 1

print("la cantidad de numeros negativos es: ", negativos)
print("la cantidad de ceros es: ", ceros)
print("la cantidad de numeros positivos es: ", positivos)

File: C++/Ejercicio 4/main.cpp
#include <iostream>

using namespace std;

int main()
{
    int numeroA, indice;
int negativos=0, ceros = 0, positivos = 0;

    for(indice = 0 ; indice< 10; indice ++)
{
cout << " ingrese un numero " << endl;
cin >> numeroA;
if (numeroA < 0)
{
negativos++ ;
}

if (numeroA == 0)
{
ceros++ ;
}

if (numeroA > 0)
{
positivos++ ;
}

}

cout << "positivos" << positivos << endl;
cout << "negativos"<< negativos<< endl;
cout << "ceros" << ceros << endl;

    return 0;
}
